src/Gui/DAGView/DAGFilter.cpp

```diff
@@ -21,15 +21,13 @@
  ***************************************************************************/
 
 #include "PreCompiled.h"
-#ifndef _PreComp_
```

Comment:
@donovaly not really a review but a question: what exactly does this do? My (un)educated guess is that certain headers are loaded sometime earlier already, and this is there in case they are not. Am I right? Are any headers included somehow also added to the list of "precompiled" headers?

Reply:
This is is used when you don't use precompiled headers (check the option containing "PCH" in CMake). With precompiled headers the headers in the if are all already included.

So with PCH one includes more header but since they are precompiled, the overall compile time is lower - for the cost of larger resulting libraries.
https://en.wikipedia.org/wiki/Precompiled_header

Reply:
I see. Thanks for clarifying.